resources/cf-url-rewriter/cf-url-rewriter.ts

```diff
@@ -66,3 +73,12 @@ function errorResponse() {
     statusDescription: 'Not found',
   };
 }
+
+function getIndexFilePath(versionNumber: string, urisplit: string[]): string {
+    if (versionNumber <= "2111") {
```

Comment:
Based on the release history https://github.com/opensearch-project/opensearch-build/releases , I see that we are still releasing `1.3.x` If we have a `1.3.15` release, looks like the logic here will still use the old index.json path.

Reply:
Just updated the requested changes. Can you please review again @tianleh. Thanks

Reply:
Commented https://github.com/opensearch-project/opensearch-ci/pull/378#discussion_r1442314387